apps/src/weblab2/helpers/aiTutorStructuredResponseHelper.ts

```diff
@@ -80,6 +84,86 @@ const getAnswerJsonSchema = (): JsonObjectSchema => {
   };
 };
 
+const getAnswerJsonSchemaAcceptReject = (): JsonObjectSchema => {
+  return {
+    type: 'object',
+    properties: {
+      tutorMode: {
+        type: 'string',
+        enum: [
+          'Build HTML',
+          'Build CSS',
+          'Build JavaScript',
+          'Ask',
+          'Hint',
+          'Debug',
+          'Explain Code',
+          'Example',
+          'Pseudocode',
+          'Documentation',
+          'Test Case',
+          'Refusal JavaScript Snippet',
+          'Refusal',
+        ],
+      },
+      goal: {
+        type: 'string',
+        description: 'What we are achieving this turn, limit to 1 line of text',
+      },
+      assumptions: {
+        type: 'string',
+        description:
+          'Explicit design choices you made from the wireframe. Format as bullets.',
+      },
+      code: {
+        type: 'array',
+        items: {
+          type: 'object',
+          properties: {
+            language: {type: 'string'},
+            sourceCode: {
+              type: 'string',
+            },
+            filename: {type: 'string'},
+          },
+          required: ['language', 'sourceCode', 'filename'],
+          additionalProperties: false,
+        },
+        description:
+          '`html`, `css`, or `js` fences. Limit to one language (html, css, or js) across the entire list. The list can be empty. Code should be formatted with appropriate newlines and indentation.  When providing modifications to student code, provide the entire contents of the file. The list can be empty. Code should be formatted with appropriate newlines and indentation. If the language is javascript or js, then the student will need to copy and paste this code into their project.',
```

Comment:
This description is slightly different from copy code version. Otherwise the schema are very similar. Can refactor to mitigate duplicate code later.

Reply:
Why does js need to be copy/pasted?